test/test_nnapi.py

```diff
@@ -32,6 +32,14 @@ def setUp(self):
         else:
             self.can_run_nnapi = False
 
+    # Created for easy override by subclasses (eg TestNnapiBackend)
+    def call_convert_to_nnapi(self, traced_module, args):
+        return convert_model_to_nnapi(traced_module, args)
+
+    # Created for subclasses to set can_run_nnapi (eg TestNnapiBackend)
+    def set_can_run_nnapi(self, bool):
```

Comment:
Don't call this argument bool, since that shadows a built-in name.  Maybe `can_run`.